packages/@glimmer/compiler/lib/compiler.ts

```diff
@@ -8,6 +8,9 @@ export interface TemplateIdFn {
 }
 
 export interface PrecompileOptions extends CompileOptions, PreprocessOptions {
+  meta?: {
```

Comment:
Answered offline, yes it can be and if so, a default module name is used instead. In the future we'll likely want to switch to using a top level `moduleName` option, and deprecate this one (left a comment to this effect).